docs/examples/partials/activity-filters.html

```diff
@@ -124,19 +124,19 @@
 			ng-if="$ctrl.size !== 'sm'"></tw-select>
 	</div>
 	<div class="col-xs-12 col-sm-6 col-md-3 col-md-pull-3 m-b-2" ng-if="$ctrl.filtersActive">
-		<div class="input-group input-group-sm">
-			<span class="input-group-addon">
-				<label class="control-label">From</label>
-			</span>
-			<input class="form-control" value="01 May 2016" />
-		</div>
+		<tw-date-lookup
+			placeholder="Start date..."
+			ng-model="$ctrl.startDate"
+			ng-max="$ctrl.endDate"
```

Comment:
ng-max for a date is probably not the most straightforward name for an interface. Could something like ng-maxDate be clearer?